gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -145,6 +146,9 @@ <h4 class="modal-title">
                     <span translate ng-if="folderEditor.trashcanClean.$error.min && folderEditor.trashcanClean.$dirty">A negative number of days doesn't make sense.</span>
                   </p>
                 </div>
+                <div class="form-group" ng-if="currentFolder.fileVersioningSelector=='systemtrash'">
+                  <p translate class="help-block">Files are moved to the system trash can when replaced or deleted by Syncthing.</p>
```

Comment:
I think the current behavior is correct (overwriting). The system only moves files to the trash when they're deleted, and a file is generally only deleted once. Using unique names if something is already in the way make sense then. 

We do it when they're overwritten and replaced as well. Frankly we maybe *shouldnt* do that for this versioning scheme, but that would make it different from all the other ones. 

Maybe we should also trash a file folderroot/foo/bar to trash/foo/bar and not trash/bar, I don't know. The latter makes more sense on a per file basis, but there's more risk for collisions.